suppressible-error-prone/src/main/java/com/palantir/suppressibleerrorprone/RemoveRolloutSuppressions.java

```diff
@@ -89,77 +83,7 @@ public Description matchAnnotation(AnnotationTree tree, VisitorState state) {
 
         String updatedText = SuppressWarningsUtils.suppressWarningsString(updatedSuppressions);
 
-        return buildDescription(tree)
-                .addFix(new LineRemovingReplacementFix(state.getSourceCode(), (DiagnosticPosition) tree, updatedText))
```

Comment:
Let's add a bit to the readme saying that we will not output nicely formatted code after this, you must use a formatter.